.storybook/preview.js

```diff
@@ -28,6 +28,8 @@ import '../assets/sass/wpdashboard.scss';
 import '../assets/sass/adminbar.scss';
 import '../assets/sass/admin.scss';
 import './assets/sass/wp-admin.scss';
+import './assets/sass/tokens.scss';
```

Comment:
Hey @derweili, thanks for making these changes. However, in my [comment](https://github.com/google/site-kit-wp/pull/6344#discussion_r1067036611) I did suggest moving these files to a `./assets/sass/stories/` subdirectory, so as to help distinguish them from the global styles. Not sure if you missed this detail or prefer to keep them directly in `sass`? Anyway, just flagging up so we can clarify this one. Admittedly there's only one additional file in there at present so it's not a big issue!

Reply:
Moved the scss files to `./assets/sass/stories/`.